compile-time error with GCC 12.2.1 on Fedora 36

**Describe the bug**
Building from source code on Fedora 36 with GCC 12.2.1 fails with the following error:

/.../rdkit/Code/GraphMol/StructChecker/Utilites.cpp: In function ‘bool RDKit::StructureCheck::lessTuple(const NbrData&, const NbrData&)’:
/.../rdkit/Code/GraphMol/StructChecker/Utilites.cpp:65:12: error: ‘const RDKit::StructureCheck::NbrData’ {aka ‘const class std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, int, int>’} has no member named ‘get’

I'm creating a PR shortly with a possible fix, using std::get.

**Configuration (please complete the following information):**
 - RDKit version: current master branch
 - OS: Fedora 36

--- a/Code/GraphMol/StructChecker/Utilites.cpp
+++ b/Code/GraphMol/StructChecker/Utilites.cpp
@@ -62,16 +62,16 @@ bool getMolAtomPoints(const ROMol &mol, std::vector<RDGeom::Point3D> &atomPoint,
 typedef std::tuple<std::string, int, int, int> NbrData;
 
 bool lessTuple(const NbrData &left, const NbrData &right) {
-  if (left.get<0>() < right.get<0>()) return true;
-  if (left.get<0>() > right.get<0>()) return false;
+  if (std::get<0>(left) < std::get<0>(right)) return true;
+  if (std::get<0>(left) > std::get<0>(right)) return false;
 
-  if (left.get<1>() < right.get<1>()) return true;
-  if (left.get<1>() > right.get<1>()) return false;
+  if (std::get<1>(left) < std::get<1>(right)) return true;
+  if (std::get<1>(left) > std::get<1>(right)) return false;
 
-  if (left.get<2>() < right.get<2>()) return true;
-  if (left.get<2>() > right.get<2>()) return false;
+  if (std::get<2>(left) < std::get<2>(right)) return true;
+  if (std::get<2>(left) > std::get<2>(right)) return false;
 
-  if (left.get<3>() < right.get<3>()) return true;
+  if (std::get<3>(left) < std::get<3>(right)) return true;
 
   return false;
 }
@@ -109,7 +109,7 @@ std::string LogNeighbourhood(
   std::sort(nbrs.begin(), nbrs.end(), lessTuple);
   for (auto &nbr : nbrs) {
     std::string bs = "";
-    switch (nbr.get<1>()) {
+    switch (std::get<1>(nbr)) {
       case Bond::SINGLE:
         bs = "-";
         break;
@@ -124,13 +124,13 @@ std::string LogNeighbourhood(
         break;
     }
     if (bs.size())
-      oss << "(" << bs << nbr.get<0>();
+      oss << "(" << bs << std::get<0>(nbr);
     else
       oss << "("
-          << "?" << (int)nbr.get<1>() << "?" << nbr.get<0>();
-    if (nbr.get<2>()) oss << (nbr.get<2>() > 0 ? "+" : "") << nbr.get<2>();
+          << "?" << (int)std::get<1>(nbr) << "?" << std::get<0>(nbr);
+    if (std::get<2>(nbr)) oss << (std::get<2>(nbr) > 0 ? "+" : "") << std::get<2>(nbr);
 
-    if (nbr.get<3>()) oss << nbr.get<3>();
+    if (std::get<3>(nbr)) oss << std::get<3>(nbr);
     oss << ")";
   }
   return oss.str();

